sencha-workspace/SlateTasksStudent/app/view/AppHeader.js

```diff
@@ -19,6 +19,42 @@ Ext.define('SlateTasksStudent.view.AppHeader', {
             align: 'center'
         },
         items: [{
+            xtype: 'combo',
+            itemId: 'student-selector',
+            name: 'StudentSelector',
+            fieldLabel: 'Student',
+            valueField: 'Username',
+            allowBlank: true,
+            queryParam: 'q',
+            margin: '0 10 0 0',
+            displayTpl: Ext.create('Ext.XTemplate',
+                '<tpl for=".">',
+                    '{FirstName} {LastName}',
+                '</tpl>'
+            ),
+            tpl: Ext.create('Ext.XTemplate',
```

Comment:
@jtwiest see above, this isn't needed if you can just produce the desired format in the model